src/main/resources/swagger/openapi.yaml

```diff
@@ -15,7 +15,18 @@ info:
     workflows, and Nextflow workflows that reference containers in formats such
     as Docker or Singularity. 
   version: 2.0.0
+  termsOfService: https://www.ga4gh.org/policies/termsandconditions.html
+  title: Dockstore Tool Registry Service
+  contact:
+    name: Dockstore@ga4gh
+    url: 'https://github.com/ga4gh/dockstore'
+    email: theglobalalliance@genomicsandhealth.org
+  license:
+    name: ' GNU Lesser General Public License'
+    url: 'https://www.gnu.org/licenses/lgpl-3.0.en.html'
```

Comment:
The TRS license is [Apache 2](https://github.com/ga4gh/tool-registry-service-schemas/blob/develop/LICENSE). Shouldn't they be the same?